packages/code-du-travail-modeles/src/__test__/3239-particuliers-employeurs-domicile/preavis-retraite/depart-mise.spec.ts

```diff
@@ -106,10 +103,10 @@ describe("Préavis de retraite de la CC 3239", () => {
       ${"Salariés du particulier employeur"}             | ${6}      | ${1}           | ${"mois"}      | ${MiseRetraiteCcReferencesParticulierEmployeurs}
       ${"Salariés du particulier employeur"}             | ${12}     | ${1}           | ${"mois"}      | ${MiseRetraiteCcReferencesParticulierEmployeurs}
       ${"Salariés du particulier employeur"}             | ${24}     | ${2}           | ${"mois"}      | ${MiseRetraiteCcReferencesParticulierEmployeurs}
-      ${"Assistants maternels du particulier employeur"} | ${2}      | ${8}           | ${"jour"}      | ${MiseRetraiteCcReferencesAssistantsMaternels}
-      ${"Assistants maternels du particulier employeur"} | ${6}      | ${1}           | ${"mois"}      | ${MiseRetraiteCcReferencesAssistantsMaternels}
-      ${"Assistants maternels du particulier employeur"} | ${12}     | ${1}           | ${"mois"}      | ${MiseRetraiteCcReferencesAssistantsMaternels}
-      ${"Assistants maternels du particulier employeur"} | ${24}     | ${2}           | ${"mois"}      | ${MiseRetraiteCcReferencesAssistantsMaternels}
+      ${"Assistants maternels du particulier employeur"} | ${2}      | ${0}           | ${"mois"}      | ${MiseRetraiteCcReferencesAssistantsMaternels}
+      ${"Assistants maternels du particulier employeur"} | ${6}      | ${0}           | ${"mois"}      | ${MiseRetraiteCcReferencesAssistantsMaternels}
+      ${"Assistants maternels du particulier employeur"} | ${12}     | ${0}           | ${"mois"}      | ${MiseRetraiteCcReferencesAssistantsMaternels}
+      ${"Assistants maternels du particulier employeur"} | ${24}     | ${0}           | ${"mois"}      | ${MiseRetraiteCcReferencesAssistantsMaternels}
```

Comment:
la règle métier à changer ? ou alors c'est qu'on l'avait pas vu dès le début ?

Reply:
Oui, un assistant maternel ne peut être mis à la retraite. Du coup, j'ai désactivé les règles et ça me retourne 0. J'aurais préféré un message indiquant que ce n'est pas possible mais je suis tombé face à un refacto trop important de notre publicodes pour le peu de gain.

Reply:
Ok c'est noté ;)